balrogscript/src/balrogscript/script.py

```diff
@@ -238,14 +239,19 @@ def get_default_config():
     server = get_task_server(task, config)
     auth0_secrets, config = update_config(config, server)
 
+    backend_version = 1
```

Comment:
Are we planning on obsoleting backend version 1 if this goes well?
If/when we make that decision, let's add a comment.

Reply:
Yep, v1 will be dying.